BATCH_RESOLVE_DOC_GENERATOR.sh

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+# BATCH_RESOLVE_DOC_GENERATOR.sh
+# Systematic resolution for DocumentGenerator.ts conflicts
+
+echo "🚀 BATCH RESOLVING DocumentGenerator.ts conflicts..."
+echo "Pattern: Preserve HEAD's safeStringify over master's basic implementation"
+
+# Apply safeStringify pattern throughout
+sed -i 's/String(error)/safeStringify(error)/g' /Users/larsartmann/projects/typespec-asyncapi/src/domain/emitter/DocumentGenerator.ts
+sed -i 's/`\${error}/`\${safeStringify(error)}/g' /Users/larsartmann/projects/typespec-asyncapi/src/domain/emitter/DocumentGenerator.ts
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)